generator/generator_tool/generator_tool.cpp

```diff
@@ -127,8 +129,8 @@ int main(int argc, char ** argv)
 
   // Load classificator only when necessary.
   if (FLAGS_make_coasts || FLAGS_generate_features || FLAGS_generate_geometry ||
-      FLAGS_generate_index || FLAGS_generate_search_index || FLAGS_calc_statistics ||
-      FLAGS_type_statistics || FLAGS_dump_types || FLAGS_dump_prefixes ||
+      FLAGS_generate_index || FLAGS_generate_search_index || FLAGS_generate_centers_table ||
```

Comment:
mpimenov <notifications@github.com> writes:

> In generator/generator_tool/generator_tool.cpp:
>
>> @@ -127,8 +129,8 @@ int main(int argc, char ** argv)
>>  
>>    // Load classificator only when necessary.
>>    if (FLAGS_make_coasts || FLAGS_generate_features || FLAGS_generate_geometry ||
>> -      FLAGS_generate_index || FLAGS_generate_search_index || FLAGS_calc_statistics ||
>> -      FLAGS_type_statistics || FLAGS_dump_types || FLAGS_dump_prefixes ||
>> +      FLAGS_generate_index || FLAGS_generate_search_index || FLAGS_generate_centers_table ||
>
> Why do you need classificator here?

Sorry, I didn't get your question. As you can see, I've just added a new
flag check to the condition.

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
>

-- 
Best regards, Yuri